s/sentry-python/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "package_name":"sentry-python",
+    "github_url":"https://github.com/getsentry/sentry-python",
+    "package_dir":"s/sentry-python/",
+    "version":"1.12.1",
+    "docker_build":false,
+     "1.12.1":{
```

Comment:
@shubham-garud Lets make it 1.12.* so upcoming version also use this script. We can revisit if there will be issues in new versions.

Reply:
@nikhil-kalbande made the necessary changes.